res/WELCOME.md

```diff
@@ -15,7 +15,17 @@ It is in general a good idea to await the automated test ("Checks") in this Pull
 
 ## Contributing the patches
 
-Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`.
+Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding comment to your PR of the form `/allow`. A good way to find other contributors is to locate recent pull requests where someone has been `/allow`ed:
```

Comment:
`by adding comment to your PR` should probably be `by adding a comment to your PR`.  Your rewording improves the original but the missing 'a' affects readability.  Sorry I didn't notice it earlier.

Reply:
Whoops! Fixed.